apoiase/app.py

```diff
@@ -20,17 +20,19 @@ def home():
     if response.status_code == 200:
         page = json.loads(response.text)
         users = page.get('supports', {}).get('users', [])
-        names = map(serializer, users)
-        names = [name for name in names if name]
+        users = filter(filter_private_supporters, users)
+        names = list(map(serializer, users))
         return jsonify(names)
 
 
-def serializer(users):
-    private=users.get('privateSupport')
-    if not private:
-        return dict(name=users.get('_id').get('name'))
+def filter_private_supporters(apoiase_users):
```

Comment:
I like the new name! So the thing is, `user` already has an boolean parameter `'privateSupport'` maybe change `private` to `is_private` can make it more clear, the thing is, this function is supposed to filter the users we get from the API. Apoia.se API returns all users, so we had to filter ourselves the ones that didn't wish to stay anonymous to display in the Serenata's website.